client/my-sites/comments/comment-list/index.jsx

```diff
@@ -55,12 +55,17 @@ export class CommentList extends Component {
 
 	state = {
 		isBulkEdit: false,
+		persistedComments: [],
+		// TODO: replace {} with [] after persistedComments is merged
```

Comment:
Is this referring to another PR, or are we splitting this work up for smaller diffs?